asapdiscovery-genetics/asapdiscovery/genetics/calculate_rmsd.py

```diff
@@ -0,0 +1,151 @@
+from pathlib import Path
+
+import numpy as np
+from asapdiscovery.data.openeye import load_openeye_pdb, save_openeye_pdb
+from asapdiscovery.modeling.modeling import superpose_molecule
+
+
+def rmsd_alignment(
+    target_pdb: str,
+    ref_pdb: str,
+    final_pdb: str,
+    target_chain="A",
+    ref_chain="A",
+):
```

Comment:
If using types should add return value also
```suggestion
) -> Tuple[float, Path]:
```
You will need `from typing import Tuple` and `from pathlib import Path`

Reply:
Thanks for the corrections! I think I will implement them locally and submit them as a separate commit. I made a few changes/additions to the files while running the job John requested and I want to avoid merge issues!

Reply:
Sounds good!